### ISSUE TITLE
Error: Error: getTranslatorId: "csljson" is not a GUID

### ISSUE THREAD

**Issue by GuillaumeFromage** [paleolimbot/rbbt#38]
Hi there,

I don't even know if this project is still active so let me know if you want me to make a pull request for this. 

Currently the default translator for a ".json" file is specified as csljson. This probably didn't work for a while, since it isn't in the list of translators used by better bibtex:
https://github.com/retorquere/zotero-better-bibtex/blob/609aa6fb0a392257bf2f8438902d46e27b6ad4e5/content/translators.ts#L109
(it probably "still worked" since the default must've been json)

However a commit from lastweek started making errors about this: https://github.com/retorquere/zotero-better-bibtex/commit/795b243619bf16aeb1e5f9bf768cc221d83cc2eb . 

A workaround for the issue is just to use "filename.lksajfd", but I don't really know why this works.

Have a nice day !

**Comment by GuillaumeFromage** [comment]
Oh, actually, the filename.lksajfd should be filename.bib .  Otherwise you have to set a default option for a translator. This way it works out of the box.

**Comment by hugo-pH** [comment]
I just came across this problem. For me writing json files has been working until today and now I got the following error messaging when running `bbt_update_bib("path_to_markdown.Rmd", path_bib = "path_to_bib.json"):

> Error: Error: getTranslatorId: "csljson" is not a GUID

Indeed, writing to `.bib` files still works. 

@GuillaumeFromage thanks for opening this issue, it pointed me to the right direction!

I fixed the issue in a local clone of the package so I could also make a pull request if necessary :)
